### ISSUE TITLE
bug: `ModuleNotFoundError: No module named 'eth_account._utils.legacy_transactions'`

### ISSUE THREAD

**Issue by sabotagebeats** [ApeWorX/ape#106]
### Environment information

```
$ pip freeze | grep ape
-e git+https://github.com/apeworx/ape-vyper@4f0a151ccadd4ab07bb0abdac3ff162494ad5e6a#egg=ape_vyper
-e git+https://github.com/apeworx/ape/@3b499d73dd8385cf29700efed991b8a84b7f26a0#egg=eth_ape
```

* Python Version: Python 3.8.10
* OS: win 10 wsl2 ubuntu 12

### What went wrong?

Please include information like:

`git pull && ape`

code that caused failure: 
https://github.com/ApeWorX/ape/commit/3b499d73dd8385cf29700efed991b8a84b7f26a0

```
$ ape
Traceback (most recent call last):
  File "/home/sabotage/.pyenv/versions/localape/bin/ape", line 33, in <module>
    sys.exit(load_entry_point('eth-ape', 'console_scripts', 'ape')())
  File "/home/sabotage/.pyenv/versions/localape/bin/ape", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/sabotage/work/ape/ape/src/ape/__init__.py", line 24, in <module>
    plugin_manager = _PluginManager()
  File "/home/sabotage/work/ape/ape/src/ape/plugins/__init__.py", line 75, in __init__
    plugin_manager.register(importlib.import_module(name))
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/sabotage/work/ape/ape/src/ape_ethereum/__init__.py", line 4, in <module>
    from .ecosystem import NETWORKS, Ethereum
  File "/home/sabotage/work/ape/ape/src/ape_ethereum/ecosystem.py", line 6, in <module>
    from eth_account._utils.legacy_transactions import (  # type: ignore
ModuleNotFoundError: No module named 'eth_account._utils.legacy_transactions'
```

### How can it be fixed?

Either I need update eth_account or this is a breaking change

**Comment by sabotagebeats** [comment]
```
$ pip install eth-ape==0.1.0a16
Collecting eth-ape==0.1.0a16
  Using cached eth_ape-0.1.0a16-py3-none-any.whl (56 kB
[truncated]
Installing collected packages: eth-ape
  Attempting uninstall: eth-ape
    Found existing installation: eth-ape 0.1.0a17.dev1+g3b499d7
    Uninstalling eth-ape-0.1.0a17.dev1+g3b499d7:
      Successfully uninstalled eth-ape-0.1.0a17.dev1+g3b499d7
Successfully installed eth-ape-0.1.0a16
$ ape --version
0.1.0a16
$ ape
WARNING: $GITHUB_ACCESS_TOKEN not set, skipping 2nd class plugins
Usage: ape [OPTIONS] COMMAND [ARGS]....
[truncated]
```

**Comment by sabotagebeats** [comment]
fixed by installing eth-account==0.5.5

```
$ pip install eth-account==0.5.5
Collecting eth-account==0.5.5
[truncated]
Installing collected packages: eth-account
  Attempting uninstall: eth-account
    Found existing installation: eth-account 0.5.4
    Uninstalling eth-account-0.5.4:
      Successfully uninstalled eth-account-0.5.4
Successfully installed eth-account-0.5.5
$ pip install -e .
[truncated]
Installing collected packages: eth-ape
  Attempting uninstall: eth-ape
    Found existing installation: eth-ape 0.1.0a16
    Uninstalling eth-ape-0.1.0a16:
      Successfully uninstalled eth-ape-0.1.0a16
  Running setup.py develop for eth-ape
Successfully installed eth-ape
$ ape
WARNING: $GITHUB_ACCESS_TOKEN not set, skipping 2nd class plugins
Usage: ape [OPTIONS] COMMAND [ARGS]...
```

**Comment by fubuloubu** [comment]
> fixed by installing eth-account==0.5.5

Yes, you have to upgrade the version. Should have updated the pin to `>=0.5.5`
